ql/lib/test/consistency/CfgConsistency.expected

```diff
@@ -0,0 +1,10 @@
+nonUniqueSetRepresentation
```

Comment:
Why is this file there? Consistency queries are not meant to be run directly, but rather through the `--consistency-queries` flag.